exatomic/va/va.py

```diff
@@ -0,0 +1,327 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015-2018, Exa Analytics Development Team
+# Distributed under the terms of the Apache License 2.0
+"""
+Vibrational Averaging
+#########################
+Collection of classes for VA program
+"""
+import numpy as np
+import pandas as pd
+import csv
+import os
+from exa.util.units import Length
+from exa.util.utility import mkp
+
+_gauss_template='''{link0}
```

Comment:
This template is code-specific. I understand that the currently available input generator doesn't meet all of the needs but regardless of that I think we should keep code-specific code in the appropriate subpackage. What do you think about migrating the generation part to the `gaussian` subpackage?

Reply:
So, like I said I did this a bit in a rush just to get something working as a draft and fine tune everything after. I was going back a lot on whether to put it in the specific quantum chemistry code package or have it here. Something I wanted to explore was to have a function where you specify the package you are using like adf or gaussian and then given the input it sends it to the respective input generator. At this moment all I can think of to do this is to have a lot of if statements which I think may be somewhat inefficient and finicky. I wanted to ask if you knew any better way to do this.

Reply:
I like the suggestion. I don't have the solution but its something that we should discuss. Maybe you can make your comment an issue and we can open up a broader discussion? For this PR, I guess lets not let the scope creep too much. One way to do that would be to put the template in the gaussian subpackage and make the template an argument in the cose below.

Reply:
I opened up an issue and put some code I was working on that would be where I'm thinking about doing this.